From a33566ad49499564dca3bcd569a9643fa521fad8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 25 Nov 2014 15:07:26 -0500
Subject: [redhat] kabi: Adding symbol find_vma (mm/nommu.c)

Message-id: <1416928046-20962-1-git-send-email-poros@redhat.com>
Patchwork-id: 100881
O-Subject: [PATCH RHEL7.1] [redhat] kabi: Adding symbol find_vma (mm/nommu.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Symbol already acked. This is repost, because checksum has been changed.
    Also added for ppc64le.

    Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Helper function to search the address space for memory
    area such that the address lies within the vm_end, used for mmap read/writes.

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_vma b/redhat/kabi/kabi-module/kabi_ppc64/find_vma
new file mode 100644
index 0000000..55c119d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_vma
@@ -0,0 +1,2 @@
+#1-
+0x7d27979f find_vma vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/find_vma b/redhat/kabi/kabi-module/kabi_ppc64le/find_vma
new file mode 100644
index 0000000..4798def
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/find_vma
@@ -0,0 +1,2 @@
+#1-
+0xd46414d6 find_vma vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/find_vma b/redhat/kabi/kabi-module/kabi_s390x/find_vma
new file mode 100644
index 0000000..2327385
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/find_vma
@@ -0,0 +1,2 @@
+#1-
+0x158b62ed find_vma vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_vma b/redhat/kabi/kabi-module/kabi_x86_64/find_vma
new file mode 100644
index 0000000..7596580
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_vma
@@ -0,0 +1,2 @@
+#1-
+0x0feb44a7 find_vma vmlinux EXPORT_SYMBOL
-- 
1.7.1